ibis/backends/sql/__init__.py

```diff
@@ -547,6 +547,21 @@ def truncate_table(
         with self._safe_raw_sql(f"TRUNCATE TABLE {ident}"):
             pass
 
+    @classmethod
+    def from_dbapi_connection(cls, con: Any, **kwargs: Any) -> BaseBackend:
```

Comment:
How about `from_connection` or `from_conn` (slight preference for the latter) so that backends that don't implement the DBAPI, like PySpark, don't have a method with a name that doesn't make sense for their implementation.

Reply:
I have a slight preference for the former, since I anticipate forgetting if it's `from_conn` or `from_con` (especially since `con = ...` is the common naming).

Reply:
Fair enough, let's go with the former then!

Reply:
Done.